styles.css

```diff
@@ -458,3 +460,399 @@ form {
 .message-label {
   padding-bottom: 40%;
 }
+
+.toolbar-links,
+.image-desktop,
+.about-project-desktop,
+.client-name-desktop,
+.name-desktop,
+.li-desktop,
+.project-desktop,
+.menu-logo {
+  display: none;
+}
+
+@media only screen and (max-width: 1440px) and (min-width: 768px) {
+  /* headliner page */
+  .toolbar {
+    padding-left: 10%;
+    width: 90vw;
+    margin-right: 0;
+  }
+
+  .toolbar-links {
+    color: #344563;
+    text-decoration: none;
+    display: flex;
+    justify-content: start;
+    width: 20vw;
+    gap: 4%;
+    margin-right: 0;
+    padding-right: 0;
+  }
+
+  .toolbar-links > a {
+    padding: 0;
+    margin: 0;
+  }
+
+  #work {
+    height: 320vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    background-color: #f7f7f9;
+  }
+
+  .wrapper-1 {
+    height: 100vh;
+    background-color: #f7f7f9;
+  }
+
+  .headline {
+    background-image: url(images/desktop-pictures/Header\ bg.png), url(images/desktop-pictures/header\ background\ shape-desktop.svg);
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: 105.9%;
+    padding: 20%;
+    padding-top: 0;
+    padding-bottom: 0;
+    border-radius: 0 0 0 180px;
+    height: 68vh;
+  }
```

Comment:
- [ ] It appears that the background image for the landing page is misaligned in your design, causing some of the images to disappear. It's important to carefully implement design details like this, as they can greatly impact the user experience. As a frontend developer, attention to detail is key to creating a polished and professional design. Please take the time to address this issue and ensure that the background image is aligned properly, so that the design functions as intended. 💪
- Figma design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/1be77c9e-c71e-4524-a712-9d36ed4b7857)

- Your design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/8724c12f-48a3-42e9-9cf4-3bf170708e38)